NodeJS-HW5/src/logger/uncaught-exception-logger.ts

```diff
@@ -0,0 +1,9 @@
+import { logger } from './logger';
+
+export const uncaughtExceptionLogger = (err: any) => {
+  const log = JSON.stringify({
+    message: 'uncaughtException',
+    error: err,
+  });
+  logger.error(log);
+};
```

Comment:
You should log and exit the process on such exception

Ref: https://stackoverflow.com/a/40867663